app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  def create
```

Comment:
There is a `cors` error when the user tries to like a post. Kindly see the error and resolve it
![likes_error](https://user-images.githubusercontent.com/35504149/173115976-b9219747-1569-49ed-8649-987d2aaea752.png)
.